book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_reader :rental
```

Comment:
- [ ] Hello! 👋 It's awesome that you're including a getter for @rentals! 📘 However, it's crucial to keep in mind that since we'll be adding a book to a rentals, it implies modifications to the rentals object. Without a setter, this could potentially lead to errors. 🚨 Let's make sure to also include a setter for @rentals to ensure smooth and error-free updates when adding a book to the rentals. 🛠️ This addition will contribute to the overall stability and functionality of our code! 💻✨

```suggestion
  attr_accessor :title, :author, :rentals
```